data/sea/54-zebra.h

```diff
@@ -11,8 +11,12 @@ typedef struct {
 
     /* outputs */
     const char  *error;
-    iint_t       fact_count;
-    iint_t       entity_count;
+    iint_t fact_count;
+    iint_t entity_count;
+
+    /* options */
+    const size_t fact_limit;
```

Comment:
yeah, this should be the alloc limit rather than fact limit, it will come later as a separate branch